connector/server/querytools/saved_cohorts.go

```diff
@@ -199,6 +221,16 @@ func RemoveCohort(userID, cohortName string) error {
 	return nil
 }
 
```

Comment:
@mickmis do we allow to perform queries like that? Aren't all queries supposed to be implemented as stored procedures?

Reply:
@f-marino you are right, queries should be stored procedures for compliance with the HUG security requirements.